REVIEW.md

```diff
@@ -0,0 +1,219 @@
+      Are there images that should be resized or optimized in another way?
+
+- [ ] User-facing changes
+    - Does the user interface or other visual elements match existing patterns in Spyder?
+      If it does not, is there a reason?
+
+Review that addresses these questions on a PR will be prefixed with `[optional]` so that the PR author(s) are clear that this feedback is not a blocker.
+Addressing this feedback is considered above-and-beyond, and may be more relevant on some PRs than others.
+PR authors may address these reviews, but are not required to do so in order for the PR to be eligible for merging.
+
+
+## For documentation-focused review
+
+### Required review
+
+The following are areas of review the community expects to be covered in the review of a documentation-focused (sometimes abbreviated as docs) PR.
+In general, reviewers need to consider whether the content is a good fit, accurate and generally clear to readers, and builds and renders with no simmediate user-visible issues.
+
+**Motivation**
+
+- [ ] Does the change improve the documentation?
+- [ ] Would it help multiple users to have this as a part of the documentation?
+- [ ] Is it appropriate for the documentation and the section it is placed in?
+
+**Content**
+
+- [ ] Is the information accurate?
+- [ ] Is the change complete, such that it won’t confuse users if it were to appear in the documentation as-is?
+- [ ] Is it clear what the text and images are trying to communicate?
+
+**Images/GIFs**
+
+- [ ] Are screenshots/GIFs reasonably legible and appropriate for the topic?
+- [ ] Are all images/GIFs scaled and optimized?
+If needed, reviewers should offer to help with this.
+
+**Technical**
+
+- [ ] Do the tests pass in the continuous integration build?
+If appropriate, help the contributor understand why tests failed.
+- [ ] Does any added/changed content render correctly, without user-visible issues?
+- [ ] Is the content free from typos and unambiguous grammar errors (review the [Style Guide](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md) for more information)?
+
+Review that addresses these questions on a PR will be prefixed with `[required]` so that the PR author(s) are clear that they are blockers.
+PR authors are responsible for addressing these reviews if prompted by the reviewer in order for that PR to be eligible for merging.
+
+### Optional review
+
+The following are themes that may be helpful to cover on a documentation-focused PR, but are optional.
+This may include feedback about non-blocking, substantive, user-visible content improvements to the text and images.
+As with all optional review, please primarily consider how a PR would impact documentation stability, clarity, and quality over aiming for subjective improvements.
+This section should be used in limited amounts to avoid overtaxing contributors, but can be used judiciously with authors who’ve successfully contributed previous pull requests to documentation repositories.
```

Comment:
I still think this divides contributors by (perceived) skill level, something that isn't (and I don't think should be) quantifiable. Is there any reason this line can't be

```suggestion
Comments under this section should be used in limited amounts to avoid overloading authors and keep the pull request focused and in scope.
```

which is the revision without the mention of a subset of contributors?

Reply:
### > I still think this divides contributors by (perceived) skill level, something that isn't (and I don't think should be) quantifiable

I'm not sure I see anywhere in my proposed revision that discusses skill level; in fact, I took pains to avoid referencing that. Rather, it simply concerns whether the contributor is newer to a repo and open source, which is readily quantifiable; not only can a reviewer's past contributions be browsed with one click to their profile, but first-time contributors to a repo get a special badge saying this on their PRs and any comment they make on them, for the [_explicit_ purpose of being extra welcoming to new contributors](https://github.blog/2017-07-25-making-it-easier-to-grow-communities-on-github/) following proven approaches of building healthy communities, while repeat contributors get a different one, and org members (core devs) get another still.

And this is all to say nothing of a reviewer's own direct personal experience of previously working with a contributor and what they responded particularly well to; for example, the most recent new contributor whose PRs I reviewed [enthusiastically _thanked_ me for providing such detailed feedback](https://github.com/spyder-ide/spyder-docs/issues/297#issuecomment-934403011) and later promised to come back again. Implying that we, as human beings, cannot take these moments of joy and appreciation (for which I live for as a reviewer) into account simply because they cannot be reduced to a simple number is something I have trouble squaring with the compassionate, humanistic virtues espoused by the code of conduct, the very principles which this document ultimately seeks to reflect.

> Is there any reason this line can't be [...] which is the revision without the mention of a subset of contributors?

Yes, I'm afraid there is, because it reflects the very motivation of this entire effort in the first place. This document already necessarily strips reviewers of much of their good-faith discretion in favor of of a more prescriptive one-size-fits-all approach, to perhaps an equal or greater degree than I've seen in practice at other source projects I've contributed to. There is a balance between to be struck between the value of carefully considered reviewer judgement and that of codifying a universal set of rules requiring all contributors to be treated the same, and I believe taking this so far as to prohibit even a mention that reviewers should take _extra_ care and compassion when working with contributors who may be newer to the project or open source not only is not helpful to our ultimate goal here, but actively harmful to it.

More broadly, I feel a rigid mandate that the work of a user who openly states they are brand new to open source making their first ever contribution to the repository must be held to the same standards as a core dev team member who is paid to work on the project and with whom I have a years-long working relationship, and require the blunt instrument of an explicit escape hatch to consider otherwise, is contrary not only to the efficient and effective functioning of the project, but is actively unhelpful toward ensuring that newer contributors feel welcome and want to continue working with the project, rather than being overwhelmed and their specific needs as an individual not cared for and understood for the sake of following strict and unyielding rules. In fact, it was the uncompromising enforcement of the latter that got us into this situation in the first place, and that careful attention to the extra care suggested by my revision above might well have avoided altogether.

That said, perhaps it is I who have taken things to far. @mrclary , I know both I and @isabela-pf have really valued your input in other discussions, and your opinions are particularly insightful and thoughtfully considered. As someone who also participates in a fair amount of review, on both sides of the coin, what is your take on this?

Reply:
Or maybe its just 4 am and I've passed the point of cogent discourse :sleepy:

Reply:
I still disagree, but you review PRs far more than me and manage the documentation/website. I'll commit this.

Reply:
I'm really sorry I went on a bit of a late night polemic (something these guidelines explicitly discourage and I should have paid more attention), and I don't want you to feel that my viewpoint should override yours simply because of my experience in one particular area.

I can understand the concerns you shared above about splitting contributors into different groups, which is why I heavily revised this section to try to eliminate that; I'm just trying to see how they still apply to a simple admonition advising reviewers to use extra restraint and care around new contributors, which I feel was the very mistake I made that resulted in this whole situation to begin with (and as a side note, as to the escape hatch, in that case I actually had asked the author for consent before reviewing, and she gave it, but that didn't prevent the worst from happening).